cloudplatform/cloudplatform-connectivity/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/DefaultHttpDestination.java

```diff
@@ -195,12 +194,8 @@ private List<Header> getHeadersFromHeaderProviders( @Nonnull final URI requestUr
         aggregatedHeaderProviders.addAll(customHeaderProviders);
         aggregatedHeaderProviders.addAll(headerProvidersFromClassLoading);
 
-        log
-            .debug(
-                "Found these {} destination header providers for a {}: {}",
-                aggregatedHeaderProviders.size(),
-                getClass().getSimpleName(),
-                Joiner.on(",").join(aggregatedHeaderProviders));
+        final String msg = "Found these {} destination header providers for a {}: {}";
+        log.debug(msg, aggregatedHeaderProviders.size(), getClass().getSimpleName(), aggregatedHeaderProviders);
```

Comment:
(minor)
AFAIK log4j can be configured how to log class names.

```suggestion
        log.debug(msg, aggregatedHeaderProviders.size(), getClass(), aggregatedHeaderProviders);
```

Reply:
I went down the `SimpleLogger` implementation of **SLF4J** and couldn't find any special handling of `Class` references.

<details>
<summary>Looks like this.</summary>

![image](https://github.com/SAP/cloud-sdk-java/assets/22489773/e1f32f82-9db5-4cd5-a2ce-ce722db9d63c)

</details>